app/src/main/java/org/fossasia/openevent/general/ticket/TicketsFragment.kt

```diff
@@ -220,15 +220,16 @@ class TicketsFragment : Fragment() {
     }
 
     private fun redirectToAttendee() {
-
         val wrappedTicketAndQty = TicketIdAndQtyWrapper(ticketIdAndQty)
         ticketsViewModel.mutableAmount.value = null
         findNavController(rootView).navigate(TicketsFragmentDirections.actionTicketsToAttendee(
             eventId = safeArgs.eventId,
             ticketIdAndQty = wrappedTicketAndQty,
             currency = safeArgs.currency,
-            amount = totalAmount
+            amount = totalAmount,
+            hasPaidTickets = ticketsViewModel.hasPaidTickets
         ))
+        ticketsViewModel.hasPaidTickets = false
```

Comment:
Why?

Reply:
If we go back and change the ticket type, then it will remain true. Another solution is to set false in on create.